modules/feedbackProducer/src/idl.thrift

```diff
@@ -73,6 +73,21 @@ service feedbackProducer_IDL
     */
    bool setTransformation(1:Matrix T_);
 
+   /**
+    * Use robot template.
+    * @param use_robot_template_ if 1 robot template is used.
+    * @param mirror_robot_template_ if 1 robot template is used.
+    * @return true/false on success/failure.
+    */
+   bool setRobotTemplate(1:i32 use_robot_template_,2:i32 mirror_robot_template_);
```

Comment:
Actually, it'd be better to use bool variables here and remove the trailing underscore.